src/metrics.cc

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+#include "metrics.h"
+
+#include <absl/strings/string_view.h>
+#include <opencensus/exporters/stats/prometheus/prometheus_exporter.h>
+#include <opencensus/stats/stats.h>
+#include <prometheus/text_serializer.h>
+
+namespace google {
+
+namespace {
+
+constexpr const char kCount[] = "1";
```

Comment:
`constexpr` implies `const` - you don't need both. :)

Reply:
`constexpr` only implies `const` for the outer type, not for the pointed-to type. So the array would have a (redundant) `const` implied by the `constexpr`, but the `char` it would point to still needs a `const`.

Reply:
Without the const, I still can't mutate the array, e.g. `kCount[0] = '2'` won't compile either way.

Reply:
by reading the thread above, the article about `constexpr` and `const` (https://medium.com/@samathy_barratt/constantly-confusing-c-const-and-constexpr-pointer-behaviour-7eddb041e15a), and the consistency of the codebase, I will use `constexpr const char`.